src/main/java/com/virtuslab/using_directives/custom/model/UsingDirectiveSyntax.java

```diff
@@ -1,9 +1,7 @@
 package com.virtuslab.using_directives.custom.model;
 
 public enum UsingDirectiveSyntax {
```

Comment:
I would get rid of this enum

Reply:
Removed and also Settings, which is no longer needed